docs/core-concepts/core-sdk/cli/dotnet-new.md

```diff
@@ -0,0 +1,47 @@
+dotnet-new
+==========
+
+## NAME
+dotnet-new -- Create a new sample .NET Core project
```

Comment:
what makes it a sample? do I have to do something different to get a real project I would want to continue using?

Just say "new .NET Core project" or "blank .NET Core project".